pages/common/smartctl.md

```diff
@@ -0,0 +1,23 @@
+# smartctl
+
+> View a disk's [SMART](https://en.wikipedia.org/wiki/S.M.A.R.T.) data and other information.
```

Comment:
This is a console tool. So `[xyz](link)` will be displayed as is. I suggest you keep the wikipedia link in a second line.